src/pds_doi_service/core/actions/test/release_test.py

```diff
@@ -169,14 +173,15 @@ def test_review_release_to_review(self):
             'no_review': False
         }
 
-        o_doi_labels = self._action.run(**release_args)
+        o_doi_label = self._action.run(**release_args)
+
+        dois, _ = DOIOstiJsonWebParser.parse_dois_from_label(o_doi_label)
 
-        for o_doi_label in o_doi_labels:
-            dois, _ = DOIOstiJsonWebParser.parse_dois_from_label(o_doi_label)
+        # Should get one DOI back with status 'review'
+        self.assertEqual(len(dois), 1)
 
-            # Should get one DOI back with status 'review'
-            self.assertEqual(len(dois), 1)
-            self.assertEqual(dois[0].status, DoiStatus.Review)
+        for doi in dois:
+            self.assertEqual(doi.status, DoiStatus.Review)
```

Comment:
Fifth time seeing this tells me this could be refactored into an abstract superclass test.

Reply:
But don't you know that unit test suites are a developer's last bastion for rampant cut-and-pasting?!
 
That being said, refactoring is a great suggestion but I may just go with helper methods internal to each test suite rather than a full blown abstract parent.

Reply:
True, true @collinss-jpl!